romana-install/roles/stack/kubernetes/install-minion/templates/cni/romana-k8s-network

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
The filename could be changed, but the contents that refer to romana-k8s-network need to remain as-is to align with the CNI plugin itself.
https://github.com/romana/kube/blob/master/CNI/romana#L39 and https://github.com/romana/kube/blob/master/CNI/romana#L78